doctr/models/recognition/master/pytorch.py

```diff
@@ -245,7 +245,7 @@ def compute_loss(
         cce[mask_2d] = 0
 
         ce_loss = cce.sum(1) / seq_len.to(dtype=torch.float32)
-        return ce_loss.unsqueeze(1)
+        return torch.mean(ce_loss)
```

Comment:
`return ce_loss.mean()`
so that we don't necessarily need to import torch for this line :+1: